extensions/jython/build.xml

```diff
@@ -48,7 +58,23 @@
             <include name="**/*.jar" />
         </fileset>
         <pathelement path="${refine.classes.dir}"/>
+        <pathelement path="${classes.dir}" />
     </path>
+
+    <path id="extension.tests.class.path">
+        <path refid="class.path"/>
+        <pathelement location="${tests.classes.dir}"/>
+        <pathelement location="${refine.tests.lib.dir}/testng-6.8.jar"/>
```

Comment:
why not the latest ?  6.10 ?

Reply:
That's what we use in OpenRefine so far - we can update it, but it's a different topic

Reply:
OK